### DISCUSSION TITLE
How to turn on lamp for custom preset TARDIS?

### DISCUSSION THREAD

**Discussion by Technoguyfication** [eccentricdevotion/TARDIS#517] [Q&A] [answered]
Hey! I'm wondering if it's possible to turn your exterior lamp on when using the custom preset. I just tested this on a fresh TARDIS, and when building a custom preset in the shell room the lamp doesn't light on the exterior.

![shell room](https://tkno.pw/3vziVp1)

![exterior](https://tkno.pw/3fZ25cs)

**Comment by eccentricdevotion** [comment, marked as answer]
The Police Box preset uses a redstone block underneath the lamp, there is no coding magic going on behind the scenes.

**Reply by Technoguyfication** [reply]
Oh my god I'm an idiot, thank you.
